### PR TITLE
Fixes #9030: We need a run with the initial promises before being able to update the promises

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9030

### CODE REVIEW

File: rudder-agent/SOURCES/Makefile [hunk 1]
```diff
@@ -129,6 +129,8 @@ install: build $(INSTALL_DEPS) install-cfengine initial-promises initial-ncf rud
 	mkdir -p $(DESTDIR)/var/rudder/cfengine-community/inputs
 	mkdir -p $(DESTDIR)/var/rudder/tmp
 	mkdir -p $(DESTDIR)/var/rudder/tools
+	-mkdir -p $(DESTDIR)/var/rudder/ncf/common
```
Comment (amousset) [inline rudder-agent/SOURCES/Makefile]: -mkdir -> mkdir

**Comment by peckpeck** [comment]
Commit modified

File: rudder-agent/debian/dirs [hunk 2]
```diff
@@ -7,7 +7,4 @@ var/rudder/cfengine-community/inputs
 var/rudder/cfengine-community/bin
 opt/rudder/etc
 var/rudder/tmp
-var/rudder/tools
```
Comment (amousset) [inline rudder-agent/debian/dirs]: Why removing it from dirs ?

Comment (peckpeck) [inline rudder-agent/debian/dirs]: because dirs is for directort creation and this creation is already done in the makefile

Comment (amousset) [inline rudder-agent/debian/dirs]: In this case we can remove almost all directories from dirs?

Comment (peckpeck) [inline rudder-agent/debian/dirs]: yes I think we should

**Comment by amousset** [comment]
Closing as the solution will be different.

**Pull request closed without merging**